content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -1,110 +1,129 @@
 
-+--------------------+--------------------------------------------------------------------------+
-| **Billing status** |                              **Conditions**                              |
-|                    +------------------------------------------------+-------------------------+
-|                    |            *On received quantities*            | *On ordered quantities* |
-+--------------------+------------------------------------------------+-------------------------+
-| *Nothing to Bill*  |       PO confirmed; no products received       |    *(Not applicable)*   |
-+--------------------+------------------------------------------------+-------------------------+
-| *Waiting Bills*    |  All/some products received; bill not created  |       PO confirmed      |
-+--------------------+------------------------------------------------+-------------------------+
-| *Fully Billed*     | All/some products received; draft bill created |    Draft bill created   |
-+--------------------+------------------------------------------------+-------------------------+
+.. note::
+   :guilabel:`3-way matching` is *only* intended to work when the bill control policy is set to
+   :guilabel:`Received quantities`.
 
-Determine when to pay with 3-way matching
-=========================================
+Pay vendor bills with 3-way matching
+------------------------------------
 
-First, go to :menuselection:`Configuration --> Settings` and activate *3-way matching*.
+When :guilabel:`3-way matching` is activated, vendor bills will display the
+:guilabel:`Should Be Paid` field under the :guilabel:`Other Info` tab. When a new vendor bill is
+created, the field will be set to :guilabel:`Yes`, since a bill can't be created until at least
+some of the products included in a purchase order have been received.
 
-.. image:: control_bills/control-bills-3-way-matching-setting.png
+.. image:: control_bills/vendor-bill-should-be-paid.png
    :align: center
-   :alt: 3-way-matching setting in Odoo Purchase
+   :alt: Draft bill should be paid field status.
 
 .. note::
-   3-way matching is intended to work with the bill control policy set to *On received quantities*.
-
-Should I pay the vendor bill?
------------------------------
+   If the total quantity of products from a purchase order has not been received, Odoo only
+   includes the products that *have* been received in the draft vendor bill.
 
-With the feature activated, your vendor bills now display the *Should Be Paid* field under the
-*Other info* tab.
+Draft bills can be edited to increase the billed quantity, change the price of the products in
+the bill, and add additional products to the bill. If this is done, the :guilabel:`Should Be Paid`
+field status will be set to :guilabel:`Exception`. This means that Odoo notices the discrepancy,
+but doesn't block the changes or display an error message, since there might be a valid reason for
```

Comment:
```suggestion
Draft bills can be edited to increase the billed quantity, change the price of the products in the
bill, and add additional products to the bill. If this is done, the :guilabel:`Should Be Paid` field
status will be set to :guilabel:`Exception`. This means that Odoo notices the discrepancy, but
doesn't block the changes or display an error message, since there might be a valid reason for
```